site/src/pages/CreateUserPage/CreateUserForm.tsx

```diff
@@ -83,14 +83,14 @@ const validationSchema = Yup.object({
 
 export const CreateUserForm: FC<
   React.PropsWithChildren<CreateUserFormProps>
-> = ({ onSubmit, onCancel, error, isLoading, myOrgId, authMethods }) => {
+> = ({ onSubmit, onCancel, error, isLoading, orgId, authMethods }) => {
   const form: FormikContextType<TypesGen.CreateUserRequest> =
     useFormik<TypesGen.CreateUserRequest>({
       initialValues: {
         email: "",
         password: "",
         username: "",
-        organization_id: myOrgId,
+        organization_id: orgId,
```

Comment:
I haven't touched this part of the codebase that much. Is `organization_id` strictly a frontend-only value, or does it eventually get sent to the server with that specific name?

If we're still using `organization_id` on the backend, I feel like it might be better to use `organizationId` everywhere, for consistency

Reply:
Ok, lets go with organizationId them